bucket/saml2aws.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/Versent/saml2aws",
+    "description": "CLI tool which enables you to login and retrieve Amazon Web Services / AWS temporary credentials using a SAML IDP.",
```

Comment:
@Ash258 I'd resist putting in a hard dependency to AWS as it may be installed in a different way (pip, choco, etc). I could add a suggest tag, but I'm not going to be near my machine for a week. What does the aws-iam-authenticator do? This only has a 64bit release, I suspect the venn intersection for SAML auth to AWS and 32bit OS is pretty small. And finally the description was taken from Chocolaty.

Reply:
@Ash258 I've made your suggested changes.